test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetUpdateCommandTests.cs

```diff
@@ -1221,5 +1221,125 @@ public class NuGetUpdateCommandTests
                 Assert.True(content1.Contains(@"<HintPath>..\..\" + customPackageFolderName + @"\a.2.0.0\lib\net45\file.dll</HintPath>"));
             }
         }
+
+        [Fact]
+        public async Task UpdateCommand_Success_Native_JS_Projects()
```

Comment:
As a recommendation, test name should follow the pattern [Given_When_Then](http://martinfowler.com/bliki/GivenWhenThen.html). 
In this case it would be named `UpdateCommand_Native_JS_Projects_Success`